_layouts/post.html

```diff
@@ -64,18 +88,19 @@ <h2 class="title" id="authors">Author{% if page.authors.size > 1 %}s{% endif %}<
                 {{site.baseurl}}/img/blank-profile.jpg
             {% endif %}
         {% endcapture %}
+
+        {% include blog-author-card.html name=author_name role=author_role img=author_img url=author_url %}
         {% if author.pronouns %}
             {% capture author_pronouns %}
                     {{author.pronouns}}
             {% endcapture %}
         {% else %}
             {% capture author_pronouns %}{% endcapture %}
         {% endif %}
-        {% include blog-author-card.html name=author_name role=author_role img=author_img url=author_url pronouns=author_pronouns %}
```

Comment:
Similarly, I think you should undo this change - you picked the wrong side of the merge conflict here (you've disabled author pronouns now)